tox.ini

```diff
@@ -27,14 +28,17 @@ changedir = .tmp/{envname}
 description =
     run tests
     devdeps: with the latest developer version of key dependencies
+    devdeps_npy2: with numpy 2.x
     cov: and test coverage
 
 deps =
     # The devdeps factor is intended to be used to install the latest
     # developer version or nightly wheel of key dependencies.
-    devdeps: pyerfa>=0.0.dev0
-    devdeps: numpy>=0.0.dev0
+    devdeps: numpy<=2.0.0.dev0
```

Comment:
What is the point of this?

Reply:
When numpy 2 is released both would pick up numpy 2. Oh... That's supposed to be a `<`

Reply:
I think what you want is oldestdeps pin to `numpy==1.*`?